keyboards/anavi/knob1/config.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 1
+
+/* Keyboard Matrix Assignments */
+// clang-format off
+#define DIRECT_PINS { \
+    { GP26 }  \
+}
+
+#define RGBLIGHT_DEFAULT_MODE 9
+
+/* Double tap reset button to enter bootloader */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+
+#define I2C_DRIVER I2CD2
+#define I2C1_SDA_PIN GP6
+#define I2C1_SCL_PIN GP7
+
+#ifdef OLED_ENABLE
+#   define OLED_DISPLAY_128X64
+#   define OLED_TIMEOUT 60000
+#   define OLED_FONT_H "keyboards/anavi/knob1/glcdfont.c"
```

Comment:
```suggestion
```

Reply:
Thank you for the code review. Done. I've forced pushed the modified commit. Thanks, Leon